internal/security/proxy/config/config.go

```diff
@@ -134,7 +135,7 @@ func (c *ConfigurationStruct) GetBootstrap() bootstrapConfig.BootstrapConfigurat
 		Host:                    c.SecretService.Server,
 		Port:                    c.SecretService.Port,
 		Path:                    c.SecretService.CertPath,
-		Protocol:                "https",
+		Protocol:                c.SecretService.Scheme,
```

Comment:
What if the config  value is blank? Default to `https` if blank?

Reply:
Nice catch, yes that would preserve existing behavior.